getting-started/kubernetes/quickstart.md

```diff
@@ -46,81 +58,44 @@ To deploy a cluster suitable for production, refer to [Calico on Kubernetes]({{
-   `kubeadm init`).
+1. Execute the following commands to configure kubectl (also returned by `kubeadm init`).
 
    ```
    mkdir -p $HOME/.kube
    sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
    sudo chown $(id -u):$(id -g) $HOME/.kube/config
    ```
 
-1. Install {{site.prodname}} with the following command.
+#### Install {{site.prodname}}
+
+1. Install the Tigera {{site.prodname}} operator and custom resource definitions.
 
    ```
-   kubectl apply -f {{ "/manifests/calico.yaml" | absolute_url }}
+   kubectl create -f {{ "/manifests/tigera-operator.yaml" | absolute_url }}
    ```
 
-   > **Note**: You can also
-   > [view the YAML in a new tab]({{ "/manifests/calico.yaml" | absolute_url }}){:target="_blank"}.
-   {: .alert .alert-info}
+1. Install {{site.prodname}} by creating the necessary custom resource. For more information on configuration options available in this manifest, see [the installation reference]({{site.baseurl}}/reference/installation/api).
 
-   You should see the following output.
-
-   ```
-   configmap "calico-config" created
-   customresourcedefinition.apiextensions.k8s.io "felixconfigurations.crd.projectcalico.org" created
-   customresourcedefinition.apiextensions.k8s.io "ipamblocks.crd.projectcalico.org" created
-   customresourcedefinition.apiextensions.k8s.io "blockaffinities.crd.projectcalico.org" created
-   customresourcedefinition.apiextensions.k8s.io "ipamhandles.crd.projectcalico.org" created
-   customresourcedefinition.apiextensions.k8s.io "bgppeers.crd.projectcalico.org" created
-   customresourcedefinition.apiextensions.k8s.io "bgpconfigurations.crd.projectcalico.org" created
-   customresourcedefinition.apiextensions.k8s.io "ippools.crd.projectcalico.org" created
-   customresourcedefinition.apiextensions.k8s.io "hostendpoints.crd.projectcalico.org" created
-   customresourcedefinition.apiextensions.k8s.io "clusterinformations.crd.projectcalico.org" created
-   customresourcedefinition.apiextensions.k8s.io "globalnetworkpolicies.crd.projectcalico.org" created
-   customresourcedefinition.apiextensions.k8s.io "globalnetworksets.crd.projectcalico.org" created
-   customresourcedefinition.apiextensions.k8s.io "networksets.crd.projectcalico.org" created
-   customresourcedefinition.apiextensions.k8s.io "networkpolicies.crd.projectcalico.org" created
-   clusterrole.rbac.authorization.k8s.io "calico-kube-controllers" created
-   clusterrolebinding.rbac.authorization.k8s.io "calico-kube-controllers" created
-   clusterrole.rbac.authorization.k8s.io "calico-node" created
-   clusterrolebinding.rbac.authorization.k8s.io "calico-node" created
-   daemonset.extensions "calico-node" created
-   serviceaccount "calico-node" created
-   deployment.extensions "calico-kube-controllers" created
-   serviceaccount "calico-kube-controllers" created
    ```
-   {: .no-select-button}
+   kubectl create -f {{ "/manifests/custom-resources.yaml" | absolute_url }}
+   ```
```

Comment:
Could we add a quick note or link to instructions on modifying `custom-resources.yaml` if they're not using the default pod cidr?